### DISCUSSION TITLE
Guidance on startDate for Dead_people_place

### DISCUSSION THREAD

**Discussion by Stephen-Gates** [muninn/graves#39] [Q&A] [answered]
The definition for [startDate](https://rdf.muninn-project.org/ontologies/graves-en.html#term_startDate) for [Dead_people_place](https://rdf.muninn-project.org/ontologies/graves-en.html#term_Dead_people_place) is not clear. 

I have a number of potential start dates for Toowong Cemetery:

- 1861 - 200 acres was set aside for cemetery purposes
- October 1870 - Cemetery Trust established
- 3 January 1871 - First burial
- 5 July 1875 - Official opened 

How can I give guidance on what date has been used? I'm leaning towards using the first burial or official opening date.

**Comment by rwarren2** [comment, marked as answer]
You can always assign any of the XML Date for any of the #Dead_people_place, #Container_for_human_remains, #Remains and #Monument classes. While prov stipulates xsd:dateTime, you can still use xsd:gYear:

```
<graves:Cemetery rdf:about="#toowong">
 <prov:startedAtTime rdf:datatype="http://www.w3.org/2001/XMLSchema#gYear">1861</prov:startedAtTime>
<graves:Cemetery>
```

I'm arguing here that the cemetery really began, as a _thing_, when the land was allocated for it. But the **Cemetery Trust** established in October 1870 is a separate legal entity that [Fibo Trust](https://spec.edmcouncil.org/fibo/ontology/BE/Trusts/Trusts/Trust) can deal with, the **first burial** on 3 January 1871 is the first grave and the **official opening** on 5 July 1875 is really an event?

All of these can be related temporally with time:before and time:after.

References move to Time/Prov Ontology #1.

**Comment by Stephen-Gates** [comment]
@rwarren2 Would you like me to convert this issue to a discussion to keep the issue list tidy?

**Comment by rwarren2** [comment]
Please. I'm about to open up a number of other issues and push changes.
